helloworld.py

```diff
@@ -1 +1,2 @@
 Hello World!
+new-greeting branch
```

Comment:
I don't really appreciate giving sys. I would be better to create a veriable from file